native/lambda_game_engine/src/game.rs

```diff
@@ -175,6 +175,40 @@ impl GameState {
         }
     }
 
+    fn activate_skills(&mut self) {
+        self.players.values_mut().for_each(|player| {
+            let skill_keys = player.skills_keys_to_execute.clone();
+            skill_keys.iter().for_each(|skill_key: &String| {
+                if let Some(skill) = player.character.clone().skills.get(skill_key) {
+                    player.add_action(
+                        Action::UsingSkill(skill_key.to_string()),
+                        skill.execution_duration_ms,
+                    );
+
+                    for mechanic in skill.mechanics.iter() {
+                        match mechanic {
+                            SkillMechanic::SimpleShoot {
+                                projectile: projectile_config,
+                            } => {
+                                let id = get_next_id(&mut self.next_id);
+
+                                let projectile = Projectile::new(
+                                    id,
+                                    player.position.clone(),
+                                    player.direction,
```

Comment:
how does this interact with the auto-aim feature?

Reply:
Hmmm I think we want it to aim to where the player is looking at. We could ask @manucamejo about this.

Reply:
I talked with manu and as for now we decided to keep it like that and iterate in the future if we need it